test/SILOptimizer/mem2reg.sil

```diff
@@ -298,7 +298,6 @@ bb2:
 
 // CHECK: bb3([[ARG:%.*]] : $@callee_owned () -> Int):
 bb3:
-// CHECK-NOT: load [[STACK]]
```

Comment:
Can you fix the other CHECK-NOT lines, too?

Reply:
I don't actually know what this is supposed to be resolved to. I see that there is a stack defined above in the `CHECK-NOT`, but that doesn't seem to define variables. Can you provide a patch of the changes you want? Thanks.

Reply:
This should be
```
// CHECK-NOT: load
```